### PR TITLE
Show all child daily attendances and reservations in the desktop UI

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

File: service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt [hunk 1]
```diff
@@ -403,30 +404,31 @@ private fun toChildDayRows(
...
+                            0
+                        }
+                    }
+                )
+
             UnitAttendanceReservations.ChildDailyRecords(
                 child = child.child,
                 dailyData =
-                    listOfNotNull(
+                    (0 until maxRecordCountOnAnyDay).mapNotNull { rowIndex ->
```
Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt:426]: `associateBy` doesn't return `null`, so this should probably be just `map`?

Comment (akheron) [reply service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt:426]: I think `0..maxRecordCountOnAnyDay` would be more idomatic Kotlin than using `until`.

**Review by akheron — COMMENTED** [review]
Zero e2e tests 😱

**Comment by akheron** [comment]
Seems to work well 🏅

File: service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt (hunk 1 above)
Comment (zch) [reply service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt:426]: `0..maxRecordCountOnAnyDay` is inclusive, while `0 until maxRecordCountOnAnyDay` is exclusive. I find using `until` to be cleaner in this case.

**Review by akheron — APPROVED** [review]

**Pull request merged**